depthai_helpers/config_manager.py

```diff
@@ -291,9 +291,9 @@ def __init__(self, args, compile_model, calc_dist_to_bb):
         # compile modules
         self.default_blob=True
         if compile_model:
-            self.blob_file, self.default_blob = self.compileBlob(self.blob_file)
+            self.blob_file, self.default_blob = self.compileBlob(self.args['cnn_model'])
```

Comment:
Does getBlobFiles serve a purpose anymore?

Reply:
I was focusing more on the fix, but yes, it can be removed.

Reply:
That's a valid point. Feel free to check it in as soon as you think it's ready then and I can do any extra cleanup and testing in a new PR.

Reply:
Actually it has a purpose, when `compile_model` is false it provides the default config.